### ISSUE TITLE
14% Finished Assignment

### ISSUE THREAD

**Issue by CajSWehst** [CajSWehst/kraken-good-accessibility#1]
@thatkidmadrid
@carolsinc

I went over the code and the accessibility, it seems to be good. I do have a small problem trying to get it to 12% it is right now at 14% so if you have any suggestions I would greatly appreciate that. 

I remember you mentioning in class to issue you about this assignment specifically because of certain aspects of the assignment.

**Comment by thatkidmadrid** [comment]
Good job! just a few points:

- I see a lot of the roles are well placed, and you're using the aria-labels properly. I do notice that you're missing some alt attribute text to describe the image.

- The header in your main can be in the same article as the rest of your content so that they don't appear as separate articles to the screen reader

Make this change and I'll add the grade to LTW

**Comment by CajSWehst** [comment]
@thatkidmadrid 
I added the suggested content. It's ready to be marked.

**Comment by thatkidmadrid** [comment]
Hi Caj, I've went over instructions in class and on brightspace for how to submit this exercise to be assessed.

I'm also comparing your output on gh-pages to the reference and there are still some issues like the header background that don't match. I also noticed that you adjusted some spacing values in your css. The only css that you would have needed to write are for the hover/focus states

Please address this and create a new Issue and I'll take another look.

Also, I encourage you to take the time to listen to your website using voiceover before submitting. Does it make sense with VoiceOver? Does it make sense actually navigating the website using VO and the keyboard?
